internal/cli/cli.go

```diff
@@ -122,3 +124,126 @@ func (opts *deleteOpts) FailMessage() string {
 	}
 	return fallbackFailMessage
 }
+
+// atlasAlertsConfigOpts contains all the information and functions to manage an alert configuration
+type atlasAlertsConfigOpts struct {
+	*globalOpts
```

Comment:
if you pass the project id, you no longer need this and can move it back, this is just to facilitate composability of the elements,
if you don't need `globalOpts` don't make others depend on it, this helps to have lighter APIs for the structs a clear use cases (single responsibility) and let's us mix and match in the future (in case we want to include `atlasAlertsConfigOpts ` but not `globalOpts`)

Reply:
I do not why with my last commit the e2e tests stop working 🤕